02-express-tutorial/answers/04-express-app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express')
+const path = require('path')
+
+const app = express()
+
+// setup static and middleware
+app.use(express.static('./public'))
```

Comment:
When serving your static files like this, what happens with the `content-type` headers we sad in part 3? Are they still served?

Reply:
It will be overridden by the default content-type headers set by the express.static middleware.